icicle/appUtils/ntt/ntt.cu

```diff
@@ -349,30 +346,44 @@ namespace ntt {
   }
 
   template <typename E, typename S>
-  cudaError_t NTT(E* input, int size, bool is_inverse, NTTConfig<S> config)
+  cudaError_t NTT(NTTConfig<E, S>* config)
   {
-    uint32_t logn = uint32_t(log(size) / log(2));
-    uint32_t n_twiddles = size; // n_twiddles is set to 4096 as BLS12_381::scalar_t::omega() is of that order.
-    size_t input_size = size * config.batch_size * sizeof(E);
-    bool is_on_device = config.are_inputs_on_device;
-    bool generate_twiddles = (config.twiddles == nullptr);
-    cudaStream_t stream = config.ctx.stream;
+    CHECK_LAST_CUDA_ERROR();
```

Comment:
Is this just for debugging now?
If not, `cudaGetLastError` resets to the latest error to `cudaSuccess`, should we use `cudaPeekAtLastError`?

Reply:
We don't really have proper error checking unfortunately, I guess this was just for debugging here
I would consider error checking out of scope for this PR tbh